FungibleToken.ts

```diff
@@ -60,24 +51,29 @@ export class FungibleToken extends TokenContract {
 
   async deploy(props: FungibleTokenDeployProps) {
     await super.deploy(props)
-
-    this.admin.set(props.admin)
-    this.decimals.set(props.decimals)
-    this.paused.set(Bool(false))
-
-    this.account.tokenSymbol.set(props.symbol)
+    this.paused.set(Bool(true))
```

Comment:
I'm just curious. Do you set the paused flag in the initialized method as well as here in deploy?

Reply:
It's probably superfluous. We urge people to use an atomic deploy, calling the `deploy()` functions of the admin and token contract, as well as `initialize()` all in one transaction. But if they don't, we at least want no transactions to happen before `initialise()` is called.